x-pack/plugins/security_solution/public/detections/pages/detection_engine/chart_panels/chart_select/translations.ts

```diff
@@ -33,24 +33,10 @@ export const CHARTS = i18n.translate('xpack.securitySolution.components.chartSel
   defaultMessage: 'Charts',
 });
 
-export const TABLE_TITLE = i18n.translate(
+export const COUNTS = i18n.translate(
   'xpack.securitySolution.components.chartSelect.tableOptionTitle',
```

Comment:
I don't know if we want to change `tableOptionTitle` to `countsOptionTitle`

Reply:
`table` is used and referenced through out these charts section, I'm more leaning totward keeping the table reference, technically `count` is the title of `table` component.